### DISCUSSION TITLE
Introductions!

### DISCUSSION THREAD

**Discussion by edakturk14** [BuidlGuidl/batch7.buidlguidl.com#1] [Announcements]
Hey everyone - welcome to Batch 7! 

My name is Eda and I've been part of the BuidlGuidl for almost 1.5 years! I'm a developer and I spend a lot of time [writing](https://mirror.xyz/edatweets.eth) about different topics/concepts in crypto. I work a lot on developer education and creating content/documentation.

Excited to have you onboard for this new batch! If you have any questions, feel free to ask anytime!

**Comment by oguzsabi** [comment]
Hi everyone,

My name is Oguz, and I've been working as a web2 developer for almost 4 years now. I enjoy gaming, but recently web3 and blockchain technologies caught my attention, with the help of a friend. Now I am trying to learn more about web3 and its decentralized nature.

I am looking forward to improving myself, and hopefully this batch will be a leap forward for me with the lucky number se7en 🤞.

**Reply by phipsae** [reply]
It's great to have you here, Oguz! I'm thrilled to see you bring your expertise into the web3 space!

**Comment by edizzum** [comment]
Hey everyone,

My name is Ediz. I started to my developer journey as C# .NET Core backend Web development. Then, started to work on AI and Machine Learning to help autonomous cars to automaticly detect lanes. Then crypto and blockchain found me and I have spent the last 2 years in this field writing smart contracts and improving myself in blockchain technology. 

Excited to be a part of this beautiful community and beside of writing smart contracts my next goal is to learn Front-end to be a Full-Stack Developer. Hopefully this batch will help me on this journey and thanks a lot to BUIDLGUIDL team :))

**Reply by phipsae** [reply]
Hi, Ediz! Really nice, you bring a lot of experience in AI and Machine Learning. I'm particularly interested in how these two fields interact. Perhaps we can discuss this further :) Looking forward to supporting you in enhancing your front-end skills!

**Comment by SimsekBerk** [comment]
I am a fourth-year Electronics and Communication student at Yildiz Technical University with
a keen interest in software development. I have experienced the Blockchain technology with
Web3 and the Smart Contract development part (e.g. Solidity, Rust, Hardhat, Web3.js,
JavaScript, TypeScript, NextJS etc...). I have participated in various projects. In my spare time,
I participate in hackathons and workshops at Blockchain events. My team and I achieved
third place at the Istanbul Blockchain Week Hackathon. In order to be more professional, I
always trying my best.

I am very excited about the adventure here. I hope we can achieve great things together. Thanks to BUIDLGUIDL !!!!

**Reply by phipsae** [reply]
Happy to have you here! You come with a lot of valuable expertise. 

Congrats to your third place in Istanbul! May I ask what your project is about?

**Comment by carletex** [comment]
Hey all!! 

My name is Carlos, I've been part of the BuidlGuidl since 2021. I've been a software developer for more than 10 years and went _onchain_ three years ago.

During this time I've built things like the [SpeedRunEthereum](https://speedrunethereum.com/) platform, the [BuidlGuidl App](https://app.buidlguidl.com/), [ABI Ninja](https://abi.ninja/) & [Scaffold-ETH 2](https://github.com/scaffold-eth/scaffold-eth-2). Inside the BuidlGuild, I also run the [Sand Garden](https://sandgarden.buidlguidl.com/).

I love learning and building things 👾

Alongside other BG members, I'll be helping you all to go through this batch. Hope you all have fun!

**Comment by Ursulovic** [comment]
Hey everyone!

My name is Ivan, and I'm a backend C# developer with great interest in blockchain technology. My journey into web3 began with the book "Mastering Bitcoin" and I have since attended several web3 events, including ETH Belgrade.

Until recently, my knowledge of blockchain and it's concepts was mostly theoretical. Now, I am looking to advance my development skills and meet people in this field.

Very excited to be here and looking forward to having a good time :)

**Reply by phipsae** [reply]
Hi Ivan! Welcome! 

It's great to have a C# backend developer interested in blockchain technology. Your expertise is definitely a valuable addition to the web3 world. Your journey from "Mastering Bitcoin" to attending ETH Belgrade sounds exciting. 

Looking forward to see what you are gonna build and for sure having a good time!

**Comment by fels21** [comment]
Hello everyone,

My name is Ivan. I have been working in web2 in the digital marketing sector for more than 5 years. 
Currently, I am learning everything related to web3 to seek opportunities in this sector. 
I came across Buildguild a few years ago, but at that time, I didn't have the knowledge and dedication necessary to continue the journey. 

When I returned to crypto a few months ago, the first community to learn from that came to my mind was Buildguild, and I was fortunate to catch this batch. I didn't think twice about participating, and here I am.

**Reply by phipsae** [reply]
Welcome back to the crypto world and BuildGuild, Ivan! 

It's great to see you giving it another try! We're happy to have you in this batch and will do our best to help you stay in web3 this time. 

Looking forward to seeing your contributions and growth here!

**Comment by phipsae** [comment]
Hi everyone,

My name is Philip, and I've been a proud member of BuidlGuidl for the past nine months. It's been a fantastic journey! I'm a developer with a passion for learning new concepts. Currently, I'm diving deep into Account Abstraction and Zero Knowledge proofs.

At BuidlGuidl, I contribute by supporting the burner wallet, PunkWallet, and the BG Nodes project, which allows you to run an Ethereum node with a single command.

I'm excited to have you here. If you have any questions, feel free to reach out to me!

Enjoy your time here! 😊

**Comment by aZrael936** [comment]
Hi everyone,
My name is Hani. have been working as a backend engineer for a year now. Got interested about web3 initially due to the trading hype. But found it every interesting with new and exciting things being build in like a daily basis. have been wondering here and there to learn more and in depth of how the things work. Excited to be learning we you.

I see we've got OG blockchain developers who have been in this space here. Extremely happy to be a part of this and also there much to be learnt.

**Comment by 3vil7win** [comment]
Greetings, frens! I'm 3vil7win and I'm a creative developer primarily interested in Three.js/WebGL and EVM/Solidity. I first started programming with Lego Dacta and my TI-83 calculator back in 2000, and have followed Bitcoin since the end of 2010. I took a long hiatus from web3 but jumped back in during the 2021 bull run mostly trading but also developing a serious itch to buidl web3 software.

I've always been a cowboy coder and don't have any formal CS training (I just love to code!), so I'm excited to connect with a cool dev community here at BuidlGuidl and Batch 7.

**Comment by GitAniket98** [comment]
Hi everyone!

I'm Aniket, a final year B.Tech-Information Technology student from India. With over a year of software development experience under my belt, I recently discovered the fascinating world of blockchain technology.  Intrigued by its potential for the future, I dived in and began learning a few months ago.

I'm thrilled to join this community and learn from all of you! I'm eager to expand my knowledge and explore the exciting possibilities that blockchain technology holds.
